app/build.gradle

```diff
@@ -210,3 +219,14 @@ tasks.register('installEmulatorRelease', Exec) {
         environment "JAVA_HOME", System.properties['java.home']
     }
 }
+
+tasks.register('clearEmulatorAppData', Exec) {
+    group("emulator")
+
+    doFirst {
+        commandLine "${project.projectDir}/development/scripts/clear_app_data.sh"
+
+        environment "ANDROID_SDK_HOME", android.sdkDirectory.absolutePath
```

Comment:
Is this the official variable name? Here, it is called `ANDROID_HOME`.

Reply:
Switched!